docs/content/dagster-cloud/developing-testing/dagster-cloud-yaml.mdx

```diff
@@ -0,0 +1,169 @@
+
+| Parameter           | Description                                            | Data type     |
+| ------------------- | ------------------------------------------------------ | ------------- |
+| `working_directory` | The path of the directory that Dagster should run from | string (path) |
+
+```yaml
+#dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: hooli_data_eng
+    working_directory: ./hooli_data_eng
+```
+
+## Build: Docker Registry
+
+Parameter reference table:
+
+| Parameter                   | Description                                                                                                                                                       | Data type                |
+| --------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------ |
+| `locations.build.directory` | The directory in your project code that you want to deploy, the default is `.`. If you have sub-folders you can specific the path to only deploy that sub-project | string (path)            |
+| `locations.build.registry`  | The Docker registry to push your code location to in the registry                                                                                                 | string (docker registry) |
+
+Single code location:
+
+```yaml
+#dagster_cloud.yaml
+
+locations:
+  - location_name: example_location
+    code_source:
+      package_name: my_dagster_project
+    build:
+      directory: ./
+      registry: 764506304434.dkr.ecr.us-west-2.amazonaws.com/branch-deployments-gh-action-test
+```
+
+Multiple code locations:
+
+```yaml
+#dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: hooli_data_eng
+    build:
+      directory: ./
+      registry: 764506304434.dkr.ecr.us-west-2.amazonaws.com/hooli-data-science-prod
+  - location_name: basics
+    code_source:
+      python_file: definitions.py
+    build:
+      directory: ./hooli_basics
+      registry: 764506304434.dkr.ecr.us-west-2.amazonaws.com/hooli-data-science-prod
+```
+
+## Python Executable
+
+By default, Dagster Cloud will use the python executable that is installed on the agent. If you want to use a different python executable, you can specify it in the dagster_cloud.yaml file. It might also be useful to have different python executables for different code locations.
+
+Parameter reference table:
+
+| Parameter                   | Description                            | Data type     |
+| --------------------------- | -------------------------------------- | ------------- |
+| `locations.executable_path` | The location of your python executable | string (path) |
+
+```yaml
+#dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: hooli_data_eng
```

Comment:
location name and package name could reflect the executable path, eg data-eng stuff vs ml stuff